.travis.yml

```diff
@@ -6,6 +6,8 @@ notifications:
     on_success: change
     on_failure: always
     template: "%{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message} | %{build_url}"
+  slack: firedrakeproject:8zAJ7fho63HNWPbKMrifGmZY
```

Comment:
Should use https://docs.travis-ci.com/user/encryption-keys/ to not share this secret info.